src/frontend/packages/lib_classroom/src/components/ClassroomWidgetProvider/widgets/DeleteClassroom/index.tsx

```diff
@@ -95,7 +100,14 @@ export const DeleteClassroom = () => {
       initialOpenValue
       title={intl.formatMessage(messages.title)}
     >
-      <Modal isOpen={isModalOpen}>
+      <Modal isOpen={isModalOpen} onClose={() => setIsModalOpen(false)}>
+        <Heading
+          size="3"
+          alignSelf="center"
+          margin={{ top: '0', bottom: 'small' }}
+        >
+          {intl.formatMessage(messages.deleteModalTitle)}
```

Comment:
```suggestion
          {intl.formatMessage(messages.deleteButtonText)}
```

Reply:
`  deleteModalTitle: {
    defaultMessage: 'Delete resource',
    description: 'Title of the delete modal.',
    id: 'components.DeleteClassroomModal.title',
  },`

Reply:
Same wondering as before, why not `Delete classroom` ?

Reply:
Same wondering as for Playlist, why not Delete classroom ?